src/ripple/app/misc/Manifest.h

```diff
@@ -257,6 +257,32 @@ class ManifestCache
     PublicKey
     getMasterKey (PublicKey const& pk) const;
 
+    /** Returns master key's current manifest sequence.
+
+        @return sequence corresponding to Master public key
```

Comment:
Micro-nit: no need for multiple **`@return`** statements:

```
@return sequence corresponding to master public key if
        configured or 0 otherwise.
```

I also appreciate the thread-safety notation, but I'm not sure that it adds a lot. Of course, I'm biased (since I understand the code) and perhaps others will find it useful.

Reply:
Merged the @return statements into one. As far as thread safety, was following the style demonstrated in the other accessor methods but removed in the followup patch.